### DISCUSSION TITLE
run multi sp-api(private developer) in same host will cause accounts related?

### DISCUSSION THREAD

**Discussion by AnsonHwang86** [amzn/selling-partner-api-models#3702] [General]
Hello, I can get refresh_token, client_id and client_secret from the developer Central in SellerCentral. and use these to get access token.
Now I want to run multi sp-api(from different selling account) in same host? Will these cause violation of section 3 of Amazon's Business Solutions Agreement? I.E. make these selling account related.

**Comment by zhakev0** [comment]
Hello. Your use case seems aligned with Business Solutions.  We would also encourage you to review Acceptable Use Policy and Data Protection Policies to ensure you are not misusing seller data, and have appropriate security controls in place:
AUP: https://sellercentral.amazon.com/mws/static/policy?documentType=AUP&locale=en_US
DPP: https://docs.developer.amazonservices.com/en_US/dev_guide/DG_DataProtectionPolicy.html
